registration/registration-test/src/main/resources/repository/UpdateAdultBio.json

```diff
@@ -35,7 +35,7 @@
 	    "language" : "fra",
 	    "value" : "Kénitra"
 	  }],
-  "dateOfBirth" : "20",
+	  "dateOfBirth" : "2010/01/02",
```

Comment:
suggesting to take age in the test data, so that you can expect definite results

Reply:
Age will get calculated from dob so I can pick the age I have changed the logic so dob can be entered rather than age.

	int age=registrationDTO.getAge();
			String group=registrationDTO.getAgeGroup();

Reply:
Updated the files now taking age instead of full dob.